frontend/dashboard/pages/Dashboard/Dashboard.tsx

```diff
@@ -108,6 +109,7 @@ export const Dashboard = ({ user, organizations, disableDebounce }: DashboardPro
                 organizations={organizations}
                 starredRepos={starredRepos}
               />
+              <ResourcesRepoList organizations={organizations} />
```

Comment:
By doing this, we are basically publishing ```resourceadm``` to production. 
We should consider adding a ```featureFlag``` until it should be published 😄